src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala

```diff
@@ -239,4 +242,12 @@ sealed trait EmbeddedKafkaSupport {
     broker.startup()
     broker
   }
+
+  def createCustomTopic(topic: String, topicConfig: Properties)(implicit config: EmbeddedKafkaConfig): Unit = {
+    val zkSessionTimeoutMs  = 10000
```

Comment:
maybe this and following `val`s should be pulled out of this method, at the trait level, in order to avoid instantation every time the method gets called